### DISCUSSION TITLE
OrchardCore module in OrchardCore.Commerce

### DISCUSSION THREAD

**Discussion by havana7** [OrchardCMS/OrchardCore.Commerce#287] [Q&A] [answered]
Hello!

I created a module in the OrchardCore solution and now I would like to use it in OrchardCore.Commerce as well.
I have imported the module and added the project reference to OrchardCore.Commerce.Web but I am getting this error.

Can you give me some solution?

Thank you.

![image](https://user-images.githubusercontent.com/394454/230440637-b22288f5-ee7f-44e0-b092-df4e2068213f.png)

**Comment by sarahelsaig** [comment, marked as answer]
The "... does not have an implementation" exception typically happens when there is a package downgrade or you try to import different versions of a project as NuGet package and as project reference. Could either of those be the case? [As a NuGet package](https://www.nuget.org/packages/OrchardCore.Commerce/#dependencies-body-tab) OrchardCore.Commerce expects the 1.5.0 version of OC.
